Ghidra/Processors/x86/data/languages/ia.sinc

```diff
@@ -1633,33 +1657,37 @@ Suffix3D: imm8        is imm8 [ suffix3D=imm8; ] { }
 
 # Instructions in alphabetical order
 
+# See 'lockable.sinc' file for instructions that are lockable
+with : lockprefx=0 {
+
```

Comment:
I had to add this `with` statement here, instead of in the `slaspec` wrapping `ia.sinc`, because I was getting errors. I believe we would need to split up this file into one that does all of the context register setup, macros, defines, etc. and one with the instruction implementations.

I think I'll let you decide what to do with that internally.